p/themes/base-theme/template.css

```diff
@@ -71,6 +71,9 @@ img.favicon {
 	width: 16px;
 	vertical-align: middle;
 }
+.feed.mute::after {
+	content: '🔇';
```

Comment:
Good idea with the mute sign. The CSS rule does not seem to work so well:

![image](https://user-images.githubusercontent.com/1008324/34470229-6bbe7460-ef2d-11e7-9a63-b2e68074b434.png)

Maybe something like:

```css
.feed.mute > .item-title::after {
	content: ' 🔇';
}
```

Reply:
Good idea, but it doesn't work in the subscription list :(

Reply:
It doesn't work either if there is an ellipsis

Reply:
I've put it before instead of after. It works everywhere.